pages/mesosphere/dcos/services/cassandra/2.7.0-3.11.4/release-notes/index.md

```diff
@@ -7,6 +7,21 @@ menuWeight: 10
 model: /mesosphere/dcos/services/cassandra/data.yml
 render: mustache
 ---
+# Release Notes for Cassandra Service version 2.7.0-3.11.4
+
+## Updates
+
+- Upgraded SDK library to version `0.57.0`
+- Added support for [Secure JMX](/mesosphere/dcos/services/{{ model.serviceName }}/2.7.0-3.11.4/advanced/#secure-jmx-enterprise)
+
+## Upgrading your cluster from 2.5.0-3.11.3 to 2.7.0-3.11.4
+
+- Run the following command to upgrade your Cassandra package: 
+  ```
+  dcos cassandra update start --package-version=2.7.0-3.11.4 --replace
+  ```
+  
+<!--
 # Release Notes for Cassandra Service version 2.6.0-3.11.4
```

Comment:
Actually, since this is the same minor version (and has a lot of content) let's keep the release notes for this version public.